Software/src/wizard/ConfigureUdpDevicePage.ui

```diff
@@ -80,18 +93,22 @@
          </property>
         </spacer>
        </item>
-       <item row="1" column="1">
-        <widget class="QLabel" name="lAddress">
+       <item row="4" column="1">
+        <widget class="QLabel" name="lPort">
          <property name="text">
-          <string>Address</string>
+          <string>Port</string>
          </property>
         </widget>
        </item>
-       <item row="3" column="1">
-        <widget class="QLineEdit" name="leAddress"/>
-       </item>
-       <item row="5" column="1">
-        <widget class="QLineEdit" name="lePort"/>
+       <item row="7" column="1">
+        <widget class="QSpinBox" name="sbTimeout">
+         <property name="maximum">
```

Comment:
should also have a minimum of `1`: it's not specified in the doc, but in practice `0` makes it not work